### ISSUE TITLE
Hidden field value does not get submitted unless Field rendered

### ISSUE THREAD

**Issue by CSFlorin** [seasonedcc/remix-forms#146]
I was very confused why this was happening for a while based on [the docs that show that declaring the hidden fields with values in the Form props is enough](https://remix-forms.seasoned.cc/examples/forms/hidden-field).

It turns out when you use a custom form, you must write an instance of Field for the hidden inputs too. You can see in [my repro](https://stackblitz.com/edit/node-w5tgiu?file=app%2Fform-action.server.ts,app%2Fform.ts,app%2Froutes%2Findex.tsx) that submitting with line 60 commented gives no value in the action for "hidden", but uncommenting it does. I wonder if this is intended behavior, and if so, if the docs can be clarified.

**Comment by darlantc** [comment]
I agree with you, this is annoying specially because it is hard to catch the problem when those hidden fields holds required values to submit the form.
It would be nice to have the hidden fields automatically pushed to the render tree for custom forms.
